README.md

```diff
@@ -12,9 +12,19 @@ To add your name read [the contributing documentation](./CONTRIBUTING.md).
 
 ---
 
+<<<<<<< HEAD
```

Comment:
There seems to be some conflict markers still present in the rebased version, but other than that the PR looks good. Thanks for attending the session.